src/prompt_constructor_multilang.py

```diff
@@ -0,0 +1,553 @@
+import os
+from .utils import read_file
+
+"""
```

Comment:
great first step, we will replace this with something a bit more modular later on!